mods/default/mapgen.lua

```diff
@@ -890,7 +890,7 @@ end
 
 -- All mapgens except mgv6
 
-function default.register_biomes(upper_limit)
+function default.register_biomes()
```

Comment:
Is there any chance this might be needed again in the future?
If so it could be kept around.

Reply:
I cannot see any benefit or reason to keep it. If such code is needed in future it can be added in future, i see no benefit in keeping it in the meantime.